### ISSUE TITLE
(3.0.53) useful to have class displayed when code pane is maximized

### ISSUE THREAD

**Issue by dalehenrich** [ericwinger/Jade#381]
![image](https://user-images.githubusercontent.com/623951/50526032-33ae2880-0a94-11e9-929b-4b15bab6e473.png)
... adding the class (or replacing the package) in the tab might be a solution ...

**Comment by LisaAlmarode** [comment]
I would like to see the current class name in the tab, if that's possible.

Also, I see what looks like artifacts in the code-pane maximized view; it's not very tidy looking.  Turns out the project-package-class-cat-method upper half is lurking there and peeking out.  

The slider that controls this is live, and I tried moving it around... but as soon as I do that, the maximize button decides it's the upper half that is under maximization control, so I can then only toggle between normal and upper-panes maximized (until I move the slider bar back up).
